mcp-server/python/crawler.py

```
@@ -0,0 +1,69 @@
+        # Save results
+        count = scraper.save_content(results, args.output_dir)
+        
+        print(f"\n✅ Successfully crawled and saved {count} documents")
+        print(f"📁 Output saved to: {args.output_dir}/")
+        print(f"📄 Combined file: {args.output_dir}/combined_crawl.md")
+        
+    except KeyboardInterrupt:
+        print("\n❌ Crawl interrupted by user")
+        sys.exit(1)
+    except Exception as e:
+        print(f"❌ Error during crawl: {str(e)}", file=sys.stderr)
```

Ruff (0.13.3)

Use explicit conversion flag

Replace with conversion flag

(RUF010)